lib/ansible/module_utils/basic.py

```diff
@@ -74,6 +74,31 @@
     imap = map                      # Python 3
 
 try:
+    basestring
+except NameError:
+    basestring = str                # Python 3
+
+try:
+    unicode
+except NameError:
+    unicode = str                   # Python 3
+
+try:
+    bytes
+except NameError:
+    bytes = str                     # Python 2
+
+try:
+    dict.iteritems
+except AttributeError:
+    def iteritems(d):               # Python 3
+        return d.items()
+else:
+    def iteritems(d):               # Python 2
+        return d.iteritems()
+
+
```

Comment:
Cosmetic: I think our coding style is to comment on a separate line instead of at the end of a line.  At this indent level it also takes fewer chars which mpdehaan was conscious about in basic.py.